x-pack/filebeat/module/cisco/asa/test/sample.log-expected.json

```diff
@@ -3589,5 +3589,69 @@
             "forwarded"
         ],
         "url.original": "http://www.example.net/images/favicon.ico"
+    },
+    {
+        "@timestamp": "2021-01-13T19:12:37.000-02:00",
+        "cisco.asa.connection_id": "27215708",
+        "cisco.asa.destination_interface": "vlan-42",
+        "cisco.asa.mapped_destination_ip": "1.2.3.4",
+        "cisco.asa.mapped_destination_port": 80,
+        "cisco.asa.mapped_source_ip": "1.2.3.4",
+        "cisco.asa.mapped_source_port": 49926,
+        "cisco.asa.message_id": "302013",
+        "cisco.asa.source_interface": "internet",
+        "cisco.asa.source_username": "LOCAL\\username",
+        "client.user.name": "username",
```

Comment:
Perhaps this should go to `user.name` instead?

Reply:
I'm not sure. Maybe this should go to both client.user.name and user.name, as well as related.user?

In my interpretation of the ECS it fits best to client.user.name since the user initiates a TCP connection:
`For TCP events, the client is the initiator of the TCP connection that sends the SYN packet(s).` [ECS docs](https://www.elastic.co/guide/en/ecs/current/ecs-client.html)

Reply:
According to https://www.elastic.co/guide/en/ecs/1.x/ecs-user-usage.html:

`In cases where a purpose-specific user field such as url.username is populated, user.name should also be populated with the same user name.`

I guess it should be copied to user.name and also kept in client.user.name

Reply:
Ok, I pushed an update. I think the new user name that is being parsed from this message is actually associated with the destination so I set `destination.user.name`. Then I set the `user.name` with that value if it wasn't already set. Then I setup related.user with `user.name` + `destination.user.name` (no duplicates).